cyber_dependabot.py

```diff
@@ -0,0 +1,80 @@
+import os
+import datetime
+import logging
+from multiprocessing import Pool
+from collections import Counter
+from typing import Iterator, Callable
+
+from addict import Dict
+import requests
+
+import storage
+import config
+
+
+def pmap(f: Callable, collection: Iterator, size: int = 10) -> list:
+    """
+    Applies `f` in parallel over `collection`.
+    Pool `size` has a sensible default of 10.
+    """
+    with Pool(size) as p:
+        return p.map(f, collection)
+
+
+def put(path: str) -> requests.models.Response:
+    """
+    Adapter of `requests.put()`, supplying github credentials.
+    """
+    ROOT_URL = "https://api.github.com"
+    headers = {
+        "Authorization": f'token {os.environ.get("TOKEN", "")}',
+        "Accept": "application/vnd.github.dorian-preview+json",
+    }
+    return requests.put(f"{ROOT_URL}{path}", headers=headers)
+
+
+def enable_alert(repo: Dict) -> int:
+    """
+    Enable vulnerability alerts on a single repo.
+    """
+
+    if os.environ.get("DRY_RUN") == "true":
+        return 200
+    else:
+        r = put(f"/repos/{repo.owner.login}/{repo.name}/vulnerability-alerts")
+        logging.info(f"repo: {repo.name}, PUT: {r.status_code}, {r.text}")
+        return r.status_code
+
+
+def enable_vulnerability_alerts() -> None:
+    """
+    Enables vulnerability alerts on every repo in repositories.json.
```

Comment:
maybe i'm missing something but how are we putting new repos in repositories.json?

Reply:
great question, it's created by the audit